crates/compiler/mono/src/layout.rs

```diff
@@ -2174,11 +2177,11 @@ macro_rules! list_element_layout {
 }
 
 pub struct Env<'a, 'b> {
-    target_info: TargetInfo,
-    arena: &'a Bump,
-    seen: Vec<'a, Variable>,
-    subs: &'b Subs,
-    cache: &'b mut LayoutCache<'a>,
+    pub(crate) target_info: TargetInfo,
+    pub(crate) arena: &'a Bump,
+    pub(crate) seen: Vec<'a, Variable>,
+    pub(crate) subs: &'b Subs,
+    pub(crate) cache: &'b mut LayoutCache<'a>,
```

Comment:
Is this only to access `subs`/`arena`? If so could we either pass those explicitly by reference where they are used or only make them public? I am particularly worried about making `seen` and `cache` visible outside of layout-generation.